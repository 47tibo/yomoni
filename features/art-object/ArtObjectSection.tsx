import { Heading } from "@/components/ui/heading";
import { Box } from "@/components/ui/box";
import { Text } from "@/components/ui/text";

type Props = {
  label: string;
  text: string;
};

function ArtObjectSection({ label, text }: Props) {
  return (
    <Box>
      <Heading size="lg">{label}</Heading>
      <Text size="lg">{text}</Text>
    </Box>
  );
}

export default ArtObjectSection;
