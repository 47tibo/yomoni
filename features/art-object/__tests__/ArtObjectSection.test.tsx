import { render } from "@testing-library/react-native";
import ArtObjectSection from "../ArtObjectSection";

describe("ArtObjectSection", () => {
  it("renders correctly with given props", () => {
    const { toJSON } = render(
      <ArtObjectSection label="Test Label" text="Test Text" />
    );
    expect(toJSON()).toMatchSnapshot();
  });
});
