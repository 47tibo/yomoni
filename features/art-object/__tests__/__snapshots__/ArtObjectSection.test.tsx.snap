// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArtObjectSection renders correctly with given props 1`] = `
<View>
  <Text
    accessibilityRole="header"
    style={
      {
        "fontSize": 32,
        "fontWeight": "bold",
        "marginVertical": 42.56,
      }
    }
  >
    Test Label
  </Text>
  <Text
    className="text-typography-700 font-body text-lg"
  >
    Test Text
  </Text>
</View>
`;
